build_tools/embed_data/generate_embed_data_main.cc

```diff
@@ -140,10 +140,11 @@ static bool GenerateImpl(const std::string& identifier,
 #define IREE_DATA_ALIGNAS_PTR __declspec(align(8))
 #else
 #include <stdalign.h>
-#define IREE_DATA_ALIGNAS_PTR alignas(alignof(void*))
+// Default set to 512b alignment.
+#define IREE_DATA_ALIGNAS_PTR alignas(64)
```

Comment:
@benvanik please let me know if we should move this define up to be
```
#if !defined(IREE_DATA_ALIGNAS_PTR)
// Default set to 512b alignment.
#define IREE_DATA_ALIGNAS_PTR alignas(64)
#if !defined(_MSC_VER)
#include <stdalign.h>
#endif  // _MSC_VER
#endif  // !IREE_DATA_ALIGNAS_PTR
```
I don't have the MSVC setup to test it out. Please advise.